atd-toolbox/data_model/python/30_make_triggers.py

```diff
@@ -18,10 +18,10 @@ def patch_template(template, table_name, pk_column_name, columns_unified):
     return (
         template.replace("$tableName$", table_name)
         .replace("$pkColumnName$", pk_column_name)
-        .replace("$affectedColumns$", ", ".join(columns_unified))
+        .replace("$affectedColumns$", ",\n        ".join(columns_unified))
         .replace(
             "$affectedColumnsWithNewPrefix$",
-            ", ".join(affected_columns_with_new_prefix),
+            ",\n        ".join(affected_columns_with_new_prefix),
```

Comment:
i also took this opportunity to add linebreaks into into the insert trigger—it makes for much better diffing.

we should lint all these migrations before we go live 👍